sample_packages/sample_python_package/README.md

```diff
@@ -0,0 +1,7 @@
+## Sample Python package
+
+This package will simulate different scenarios to test package analysis on.
+
+To use this package for local analysis, build this package by running
+`python3 -m build` in this directory. The package will be located in the dist/
```

Comment:
Does the build step require any dependencies?

Reply:
The developer won't need to install any dependencies themselves on their local system since I've used pyproject.toml. An isloated environment will be used to build the package and relevant dependencies will be downloaded during that process, so just running the single build command should work.